pyfstat/core.py

```diff
@@ -1009,20 +1010,22 @@ def calculate_twoF_cumulative(
         return taus, np.array(twoFs)
 
     def _calculate_predict_fstat_cumulative(
-        self, N, label=None, outdir=None, IFO=None, pfs_input=None
+        self, N, label=None, outdir=None, pfs_input=None
     ):
         """Calculates the predicted 2F and standard deviation cumulatively
```

Comment:
This explanation (and for the other related functions) should be extended, since it wasn't enough for us to understand what they do without detailed code vivisection. ;) But I can take a turn on that.